book/content/documentation/documenation.md

```diff
@@ -0,0 +1,50 @@
+# Documentation
+
+## About
+Both the project and the source code should be documentated in order for others to understand what the project is about, how they can use it and how they can contribute to it.
+
+### Markdown
+[Markdown](https://en.wikipedia.org/wiki/Markdown) is a lightweight markup language with plain-text-formatting syntax. Its design allows it to be converted to many output formats, but the original tool by the same name only supports HTML. Markdown is often used to format readme files, for writing messages in online discussion forums, and to create rich text using a plain text editor.
+
+GitHub and other project and code hosting services automatically render markdown files for humans to read. 
+
+You can learn about markdown syntax and how to master this language in an excellent GitHug [Mastering markdown Guide](https://guides.github.com/features/mastering-markdown/). Try it out with an [online editor](https://markdown-it.github.io)
+
+## Project Documentation
+Describe your project for others to understand what it is about, what is the audience, how others can contribute to it. The [Mozilla Open Leadership Training Series](https://mozilla.github.io/open-leadership-training-series/) [Opening Your Project Chapter](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/) is a good starting point to learn about how to efectively communicate information abou your project.
```

Comment:
Italics, bold text or simialr should be used to indicate that `Opening Your Project Chapter` is a different link from _Mozilla OpenLeadership Training Series_